### PR TITLE
Fix responsive hotspots on Android

### PR DESCRIPTION
**Pull request by Getitdan**
Hotspots work perfectly fine on Desktop & iOS (Safari / Chrome). However, hotspots aren't responsive in Android at all.

Fixes issue #62.

### CODE REVIEW

File: src/Viewer.js [hunk 1]
```diff
@@ -136,9 +138,14 @@ function Viewer(domElement, opts) {
   setFullSize(this._controlContainer);
 
   // Prevent bounce scroll effect on iOS.
-  this._controlContainer.addEventListener('touchstart', function(event) {
-    event.preventDefault();
-  });
+  // Applied only for iOS, as Android's events must have the default action to allow interaction with hotspots.
+  var userAgent = navigator.userAgent || navigator.vendor || window.opera;
```
Comment (tjgq) [inline src/Viewer.js]: Please remove this line since it's no longer required.

Comment (Getitdan) [inline src/Viewer.js]: Removed, thanks.

**Comment by tjgq** [comment]
Thanks for fixing this!

**Comment by Getitdan** [comment]
Thank you for maintaining this awesome library!

**Pull request merged**